Sources/SwiftIfConfig/IfConfigError.swift

```diff
@@ -12,6 +12,7 @@
 
 import SwiftDiagnostics
 import SwiftSyntax
+import SwiftSyntaxBuilder
 
 /// Describes the kinds of errors that can occur when processing #if conditions.
 enum IfConfigError: Error, CustomStringConvertible {
```

Comment:
Since `IfConfigError` can also represent warnings now, I think a better name would be `IfConfigDiagnostic`.

Reply:
Good idea! Done in https://github.com/swiftlang/swift-syntax/pull/2806